doc/release-notes/5.1.1-release-notes.md

```diff
@@ -0,0 +1,59 @@
+# Dataverse 5.1.1
+
+This minor release adds important optimizations to installations running on AWS S3. It is recommended that 5.1.1 be used in production instead of 5.1.
+
+## Release Highlights
+
+### Connection Pool Size Configuration Option, Connection Optimizations
+
+This release adds optimizations around closing streams and channels that may hold S3 http connections open and exhaust the connection pool. In parallel, this release adds the ability to increase the size of the connection pool, so a larger pool can be configured if needed.
```

Comment:
Might want to say 'Dataverse 5.1 improved the efficiency of making S3 connections through use of an http connection pool.' 
And also add to the last sentence - 'In parallel, this release increases the default pool size from 50 to 256 and adds the ability...;

Reply:
Good stuff @qqmyers... added in https://github.com/IQSS/dataverse/pull/7319/commits/d61b5933a2a8206695e72668d132a0a36c2f7a3e